app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -26,9 +27,9 @@ const typeOfAssistance = [
   'Q Foundation: Person with Disability Rent Subsidy',
   'Q Foundation: Senior Rent Subsidy',
   'San Francisco AIDS Foundation',
-  'Section 8 HCV (Tenant) Voucher',
+  'Section 8 HCV (tenant) voucher',
   'Self-Help for the Elderly',
-  'VASH Voucher',
+  'VASH voucher',
```

Comment:
@jrubenoff I've changed the capitalization on these two options back to their original Salesforce picklist capitalizations, so that they will exactly match the picklist values.

Reply:
That's unfortunate, but OK!